### PR TITLE
Uhf 6325 chat via leijuke

### PR DESCRIPTION
**Pull request by Arskiainen**
# [UHF-6325](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6325)

Replaces chat blocks with leijuke.
## What was done

* Updated platform config and theme.
* Replaced chat blocks with leijuke blocks.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git pull origin UHF-6325-chat-via-leijuke`
  * `make fresh`
* Run `make drush-cr`

## How to test

* [ ] Check that this feature works by visiting /pysakointi page and interacting with the chat leijuke.
* [ ] Check that code follows our standards

## Designers review

* [X] This PR does not need designers review
* [ ] This PR has been visually reviewed by a designer (Name of the designer)

## Other PRs

* Link to other PR

### CODE REVIEW

File: conf/cmi/core.entity_view_display.paragraph.target_group_link_item.default.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-uuid: 5c07c500-20d9-4232-a1f1-b83c57fa1a53
```
Comment (teroelonen) [inline conf/cmi/core.entity_view_display.paragraph.target_group_link_item.default.yml:1]: Why are these uuid:s being removed? Should they be?

File: conf/cmi/eu_cookie_compliance.cookie_category.essential.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ _core:
...
...>\r\n\t\t\t</td>\r\n\t\t\t<td>10950 days</td>\r\n\t\t\t<td>&nbsp;</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>frosmo_quickContext</td>\r\n\t\t\t<td>hel.fi</td>\r\n\t\t\t<td>\r\n\t\t\t<p>The Frosmo cookie improves the user experience and provides targeted content.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>365 days</td>\r\n\t\t\t<td>&nbsp;</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>id</td>\r\n\t\t\t<td>frosmo.com</td>\r\n\t\t\t<td>\r\n\t\t\t<p>The cookie facilitates visiting the website but does not save information that could be used to identify the visitor.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>5947 days, 15 hours</td>\r\n\t\t\t<td>Third party</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>JSESSIONID</td>\r\n\t\t\t<td>helsinkikanava.fi</td>\r\n\t\t\t<td>\r\n\t\t\t<p>The cookie is an obligatory cookie that facilitates visiting the website.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>Session</td>\r\n\t\t\t<td>Third party</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>COOKIE_SUPPORT</td>\r\n\t\t\t<td>helsinkikanava.fi</td>\r\n\t\t\t<td>\r\n\t\t\t<p>The cookie facilitates managing cookies on the website.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>365 days</td>\r\n\t\t\t<td>Third party</td>\r\n\t\t</tr>\r\n\t</tbody>\r\n</table>\r\n"
```
Comment (teroelonen) [inline conf/cmi/eu_cookie_compliance.cookie_category.essential.yml:10]: Quite a lot of text being removed - is this a change that we want to have?

File: conf/cmi/language/fi/eu_cookie_compliance.cookie_category.essential.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
...d>&nbsp;</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>frosmo_quickContext</td>\r\n\t\t\t<td>hel.fi</td>\r\n\t\t\t<td>\r\n\t\t\t<p>Frosmon eväste käyttökokemuksen parantamiseksi ja kohdennettujen sisältöjen tarjoamiseksi.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>365 päivää</td>\r\n\t\t\t<td>&nbsp;</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>id</td>\r\n\t\t\t<td>frosmo.com</td>\r\n\t\t\t<td>\r\n\t\t\t<p>Käytetään yksilöimään kävijä (selain) eri domaineissa ja verkkosivuilla</p>\r\n\t\t\t</td>\r\n\t\t\t<td>5947 päivää, 15 tuntia</td>\r\n\t\t\t<td>Kolmas osapuoli</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>JSESSIONID</td>\r\n\t\t\t<td>\r\n\t\t\t<p>helsinkikanava.fi</p>\r\n\r\n\t\t\t<p>coh-chat-app-prod.eu-de.mybluemix.net</p>\r\n\t\t\t</td>\r\n\t\t\t<td>\r\n\t\t\t<p>Sivuston pakollinen eväste mahdollistaa kävijän vierailun sivustolla.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>Istunto</td>\r\n\t\t\t<td>Kolmas osapuoli</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>COOKIE_SUPPORT</td>\r\n\t\t\t<td>helsinkikanava.fi</td>\r\n\t\t\t<td>\r\n\t\t\t<p>Mahdollistaa evästeiden hallinnan sivustolla.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>365 päivää</td>\r\n\t\t\t<td>Kolmas osapuoli</td>\r\n\t\t</tr>\r\n\t</tbody>\r\n</table>\r\n"
```
Comment (teroelonen) [inline conf/cmi/language/fi/eu_cookie_compliance.cookie_category.essential.yml:2]: Quite a lot of text being removed - is this a change that we want to have?

File: conf/cmi/language/sv/eu_cookie_compliance.cookie_category.essential.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
...
...>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>frosmo_quickContext</td>\r\n\t\t\t<td>hel.fi</td>\r\n\t\t\t<td>\r\n\t\t\t<p>Frosmos cookie förbättrar användarupplevelsen och erbjuder inriktat innehåll.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>365 dagar</td>\r\n\t\t\t<td>&nbsp;</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>id</td>\r\n\t\t\t<td>frosmo.com</td>\r\n\t\t\t<td>\r\n\t\t\t<p>Cookie möjliggör besöket på webbplatsen men sparar inga uppgifter som kan användas för att identifiera besökaren.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>5947 dagar, 15 timmar</td>\r\n\t\t\t<td>Tredje part</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>JSESSIONID</td>\r\n\t\t\t<td>\r\n\t\t\t<p>helsinkikanava.fi</p>\r\n\r\n\t\t\t<p>coh-chat-app-prod.eu-de.mybluemix.net</p>\r\n\t\t\t</td>\r\n\t\t\t<td>\r\n\t\t\t<p>Kakan är en obligatorisk kaka som gör det möjligt för besökaren att besöka webbplatsen.</p>\r\n\t\t\t</td>\r\n\t\t\t<td>session</td>\r\n\t\t\t<td>Tredje part</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>COOKIE_SUPPORT</td>\r\n\t\t\t<td>helsinkikanava.fi</td>\r\n\t\t\t<td>Kakan möjliggör hanteringen av kakor på webbplatsen.</td>\r\n\t\t\t<td>365 dagar</td>\r\n\t\t\t<td>Tredje part</td>\r\n\t\t</tr>\r\n\t</tbody>\r\n</table>\r\n"
```
Comment (teroelonen) [inline conf/cmi/language/sv/eu_cookie_compliance.cookie_category.essential.yml:2]: Quite a lot of text being removed - is this a change that we want to have?

**Review by teroelonen — CHANGES_REQUESTED** [review]
There is some configuration changes that seem a bit off so if you could check those 🙇

File: conf/cmi/core.entity_view_display.paragraph.target_group_link_item.default.yml (hunk 1 above)
Comment (Arskiainen) [reply conf/cmi/core.entity_view_display.paragraph.target_group_link_item.default.yml:1]: These are related to some other update from platform config iirc.

File: conf/cmi/eu_cookie_compliance.cookie_category.essential.yml (hunk 2 above)
Comment (Arskiainen) [reply conf/cmi/eu_cookie_compliance.cookie_category.essential.yml:10]: The only thing the above should change is remove the chat related cookies description from the essential group. Those are in their own group. This is also formatted text so it's near impossible to deduce what is going on without checking the before and after in the cookie information page.

**Comment by Arskiainen** [comment]
made a new clean pr https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/478

**Pull request closed without merging**